client/components/BasicInformation.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import ContactAgent from './ContactAgent.jsx';
+import MapModule from './MapModule.jsx';
+import {ContentOuterContainer, ContentInnerContainerRow, BasicInformationGrid, BasicInformationSectionOne, BasicInformationSectionTwo, BasicInformationFormSection, BasicInformationLocalSection, FontAwesomeIcon, PrimaryButton } from '../styles.js'
```

Comment:
Is this your preferred method of using styled components instead of having them individually on each components file?

Reply:
I'm not entirely sure yet. It is nice to have a library of components that can be shared, so I'm leaning towards it.